de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/DesktopMessages.properties

```diff
@@ -41,3 +41,24 @@ sectionOne = Section 1
 resetHpc = Reset HPC Token
 resetHpcPrompt = <p>Click `Reset HPC Token` to reset your HPC token:</p>
 hpcResetFailure = Unable to reset your HPC token. Please contact support!
+webhooks=Webhooks
+webhooksPrompt = Add incoming webhook integration for slack. DE will post notifications to this URL:
```

Comment:
Minor, but we might wanna modify this to "...integration for Slack. The DE will..."